tests/integration/api/test_activation.py

```diff
@@ -40,6 +40,16 @@
     "project_id": 1,
 }
 
+TEST_ACTIVATION_INSTANCE = {
+    "name": "test-activation_instance",
+    "rulebook_id": 1,
+    "inventory_id": 1,
+    "extra_var_id": 1,
+    "working_directory": "/tmp",
+    "execution_environment": "quay.io/aizquier/ansible-rulebook",
```

Comment:
Just a quick question on my end. Is there any sort of generic execution environment variable we could set up for testing, or is this the standard environment we have decided as a testing execution environment?

Reply:
@shannon-donahue No, there is no such a standard execution environment. For testing, it can be any variable name.

Reply:
This is the execution environment by default, which is an image automatically built from the official repo. 
https://github.com/ansible/eda-server/blob/main/src/eda_server/schema/activation.py#L36

Reply:
Thank you both for the clarification